docs/content/any/project/changelogs/NEXT.md

```diff
@@ -8,9 +8,9 @@ description: >-
 draft: true
 ---
 
-## `RELEASED_PACKAGE_1` NEW_VERSION
+## `0.26.3` NEW_VERSION
 
-### LANGUAGE (e.g., 'Core' or 'Python' or 'Node.js')
+### Node.js (e.g., 'Core' or 'Python' or 'Node.js')
 
 #### Breaking changes
```

Comment:
I think this is technically a breaking change, but it's fixing something that was broken.

Reply:
So not sure if it needs a version bump.